test/functional/tests/common/lvm_common_utils.py

```diff
@@ -29,7 +30,8 @@ def run_fio_on_lvm(volumes: []):
 
 
 def get_test_configuration():
-    config = InitConfig.create_init_config_from_running_configuration()
+    InitConfig.create_init_config_from_running_configuration()
+    config_output = TestRun.executor.run(f"cat {opencas_conf_path}")
     devices = get_block_devices_list()
 
-    return config, devices
+    return config_output.stdout, devices
```

Comment:
Actually this isn't related to LVM either. Maybe it should be moved to init_config.py?

Reply:
That's true but it's used only by lvm related tests so looked like good "common place". I would say - if, in the future, someone would like to use it it can be moved of course:)

Reply:
file lvm_common_utils.py will be removed and it's functions placed within corresponding tests:)